distributions/openhab/src/main/resources/bin/update.lst

```diff
@@ -46,6 +46,9 @@ DEFAULT;$OPENHAB_USERDATA/etc/org.ops4j.pax.logging.cfg
 DEFAULT;$OPENHAB_USERDATA/etc/log4j2.xml
 DEFAULT;$OPENHAB_USERDATA/etc/org.ops4j.pax.logging.cfg
 
+[3.1.0]
```

Comment:
I'd think this sections needs to be moved to https://github.com/openhab/openhab-distro/pull/1247/files#diff-8be360a52adcae6cc237c7d0e4658b2d50028192596d6489f64ee61e4d43f4aaR40, i.e. under `[[MSG]]` and not under `[[PRE]]`.